pkg/bundle/internal/truststore/types_test.go

```diff
@@ -23,12 +23,65 @@ import (
 	"testing"
 
 	"github.com/pavlo-v-chernykh/keystore-go/v4"
+	"github.com/stretchr/testify/assert"
 
 	"github.com/cert-manager/trust-manager/pkg/apis/trust/v1alpha1"
 	"github.com/cert-manager/trust-manager/pkg/util"
 	"github.com/cert-manager/trust-manager/test/dummy"
 )
 
+func Test_Encoder_Deterministic(t *testing.T) {
+	tests := map[string]struct {
+		encoder             Encoder
+		expNonDeterministic bool
+	}{
+		"JKS default password": {
+			encoder: NewJKSEncoder(v1alpha1.DefaultJKSPassword),
+		},
+		"JKS custom password": {
+			encoder: NewJKSEncoder("my-password"),
+		},
+		"PKCS default password": {
+			encoder: NewPKCS12Encoder(v1alpha1.DefaultPKCS12Password),
+		},
+		"PKCS custom password": {
```

Comment:
nitpick: There are other things called "PKCS", could we explicitly write `PKCS#12` in these test names instead?

Reply:
Go catch, updated!